MoltenVK/MoltenVK/GPUObjects/MVKDevice.mm

```diff
@@ -747,11 +779,11 @@
 			case VK_STRUCTURE_TYPE_PHYSICAL_DEVICE_SAMPLE_LOCATIONS_PROPERTIES_EXT: {
 				auto* sampLocnProps = (VkPhysicalDeviceSampleLocationsPropertiesEXT*)next;
 				sampLocnProps->sampleLocationSampleCounts = _metalFeatures.supportedSampleCounts;
-				sampLocnProps->maxSampleLocationGridSize = kMetalSamplePositionGridSize;
-				sampLocnProps->sampleLocationCoordinateRange[0] = 0.0;
-				sampLocnProps->sampleLocationCoordinateRange[1] = (15.0 / 16.0);
+				sampLocnProps->maxSampleLocationGridSize = kMetalSampleLocationGridSize;
+				sampLocnProps->sampleLocationCoordinateRange[0] = kMVKMinSampleLocation;
+				sampLocnProps->sampleLocationCoordinateRange[1] = kMVKMaxSampleLocation;
 				sampLocnProps->sampleLocationSubPixelBits = 4;
```

Comment:
It should be possible to derive this value from the constants above as well--4 is the binary logarithm of 16, because 2<sup>4</sup> = 16.

Reply:
Ah. Nice idea. Done.